stdlib/public/core/StaticString.swift

```diff
@@ -51,16 +95,14 @@ public struct StaticString
   /// - bit 0: set to 0 if `_startPtrOrData` is a pointer, or to 1 if it is a
   ///   Unicode scalar.
   ///
-  /// - bit 1: set to 1 if `_startPtrOrData` is a pointer and string data is
-  ///   ASCII.
+  /// - bit 1: set to 1 if `_startPtrOrData` is an ASCII scalar or sequence.
```

Comment:
This seems less clear than what was there before — was the old documentation incorrect?

Reply:
Yes, the old documentation was outdated. Swift 1.0 didn't set bit 1 when `unicodeScalar.isASCII`, but this was fixed and tested since Swift 1.1 (in commit 4f38403edaecb11b2102a654be98d45f83e25ea1). Please suggest an improved comment for `_flags` (and `isASCII`).

Reply:
I've updated `_flags` and `isASCII` in 10018e30e46ef7637454c58c1bf503d6649e42e9 (and then I've squashed all the commits).